.gitignore

```diff
@@ -5,3 +5,4 @@
 .DS_Store
 venv/
 .pytest_cache/
+__pycache__/
```

Comment:
Do not add a new directory in the `.gitignore` file here. Remove this change from your PR.